hack/verify-operator-sdk-gen.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+# HEKETI-SKIP-SHELLCHECK 
```

Comment:
Why is this file here?

Reply:
@davidvossel created I don't know the exact purpose